eo-runtime/src/test/java/org/eolang/PhDefaultTest.java

```diff
@@ -17,7 +17,7 @@
  * Test case for {@link PhDefault}.
  * @since 0.1
  */
-@SuppressWarnings("PMD.TooManyMethods")
+@SuppressWarnings({"PMD.TooManyMethods", "PMD.AvoidGodClass", "PMD.GodClass"})
```

Comment:
@karishka1222 in PMD, it looks like that we don't have `PMD.AvoidGodClass` rule, so let's remove its suppression, since it has no effect